apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -58,6 +58,10 @@ def get_binding
     # @return [String] error message
     attr_reader :render_info_view_error_message
 
+    # Application parameters provided by the user that will be displayed in the session card.
+    # @return [Hash] application parameters
+    attr_accessor :parameters
```

Comment:
```suggestion
    attr_accessor :display_choices
```

I'd say this could be re-worded, though I'm not sure if  `display_choices` is the best.

Reply:
Updating....